pysoa/server/server.py

```diff
@@ -255,11 +266,11 @@ def process_job(self, job_request):
             # Send an error response if no middleware caught this.
             # Formatting the error might itself error, so try to catch that
             self.metrics.counter('server.error.unhandled_error').increment()
-            return self.handle_error(e)
+            return self.handle_exception(e)
 
         return job_response
 
-    def handle_error(self, error, variables=None):
+    def handle_exception(self, error, variables=None):
```

Comment:
Is it better to keep this method name unchanged? Also name the new method some other name other than `*error` or `*exception`, as those two words interleave and confusing.

Reply:
I renamed that method specifically because it was ambiguous, but I just ended up with two ambiguous method names. I'll improve further.

Reply:
How does it look now, Yan?